lib/taskana-core/src/main/java/pro/taskana/common/internal/util/DaysToWorkingDaysConverter.java

```diff
@@ -74,10 +74,16 @@ public static void setCustomHolidays(List<LocalDate> holidays) {
   }
 
   public long convertWorkingDaysToDays(Instant startTime, long numberOfDays) {
+    if (startTime == null) {
```

Comment:
How can it be that the startTime is null?
That parameter is necessary. - Should we throw an exception instead of going to the default?

Reply:
bump

Reply:
I'll throw an exception if startTime is null.
What do you mean by 'bump' ?